test/test_vcf.py

```diff
@@ -12,33 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from varcode import load_variants
+from varcode import load_vcf
 
 VCF_FILENAME = "data/somatic_hg19_14muts.vcf"
 
 def test_vcf_reference_name():
-    variants = load_variants(VCF_FILENAME)
-    # the raw reference name can be a file path to the hg19 FASTA file
-    assert variants.reference_path and "hg19" in variants.reference_path, \
-        "Expected hg19 reference, got %s" % (variants.reference_path,)
+    variants = load_vcf(VCF_FILENAME)
     # after normalization, hg19 should be remapped to GRCh37
-    assert variants.reference_name == "GRCh37"
+    assert variants.reference_names() ==  { "GRCh37" }
 
 def test_vcf_number_entries():
     # there are 14 mutations listed in the VCF, make sure they are all parsed
-    variants = load_variants(VCF_FILENAME)
+    variants = load_vcf(VCF_FILENAME)
```

Comment:
You can use `nose.tools.eq_` but sometimes I find it more informative to have a string explaining where the value came from.